blockbase/inc/disable-site-editor.php

```diff
@@ -19,27 +29,21 @@ function add_disable_site_editor_setting() {
 			'id'    => 'universal-theme-disable-site-editor',
```

Comment:
I know this is unrelated to the PR (it happened in #4219), but it's breaking my brain 😅 

```
'label' => __( 'Enable Site Editor', 'gutenberg' ),
'id' => 'universal-theme-disable-site-editor',
```

Could we update the option id to `universal-theme-enable-site-editor` instead of `disable`?

Reply:
yeah good catch that's me being sloppy....

Reply:
Updated this, and renamed the file for clarity too.

Reply:
Thanks! ✨